pt/lessons/basics/modules.md

```diff
@@ -102,6 +102,35 @@ iex> %{name: "Sean"} = sean
 #Example.User<name: "Sean", roles: [...], ...>
 ```
 
+A partir do Elixir 1.8, structs incluem introspecção customizáveis. Para entender o que isso significa e como devemos usar, vamos inspecionar nossa captura `sean`:
```

Comment:
Could you please, update to have each phrase on its own line? Related #1652.

Reply:
This new rule will be applied to all the `pt` files in the future?

Reply:
Yes, probably @ogabriel. I'm not sure if we need to update everything to follow the guideline, but we can update whenever we update a translation.